wikispeedruns/auth/passwords.py

```diff
@@ -25,7 +25,7 @@ def check_password(user: dict, password: str) -> bool:
     id = user["user_id"]
     hash = user["hash"]
 
-    if (user["is_old_hash"]):
+    if "is_old_hash" in user and user["is_old_hash"]:
```

Comment:
Why would this not be in user? It should always be queried?

Reply:
<img width="808" alt="Screen Shot 2022-02-24 at 7 31 32 PM" src="https://user-images.githubusercontent.com/17424008/155648525-3fdebfb7-a6b3-484d-afe6-0311fdfe9d99.png">

this was the error I was trying to fix - not sure if it's just an issue locally or not.

I was trying to log into a user that was created before the recent password changes

Reply:
can you add the full stack trace?

Reply:
<img width="968" alt="Screen Shot 2022-02-24 at 9 18 57 PM" src="https://user-images.githubusercontent.com/17424008/155658457-690ce427-ce5d-4d36-af78-f7aae1f9c4cb.png">

Repro:
on recent pull of main branch, no local changes
attempt to login to account on local instance

Reply:
nvm, solution is to re-run createDB script